packages/auth/addon/components/spree-auth.js

```diff
@@ -11,8 +11,9 @@ import layout from '../templates/components/spree-auth';
   ember g spree-ember-auth-component
   ```
 
-  This will install a layout file at: `app/templates/components/spree-auth`, and
-  a component file at `app/components/spree-auth`, ready to be extended.
+  This will install all of the Spree Ember Auth component files into your
+  host application at `app/components/spree-*.js`, ready to be extended or
+  overriden.
```

Comment:
Can you switch this back to the old description?  might as well merge this file though, as it strips a bunch of whitespace.

Need to figure out how to automatically strip whitespace in Vim... I just started using it about a month ago.